apps/backend/src/modules/contributors/contributor.controller.ts

```diff
@@ -167,22 +188,23 @@ class ContributorController extends BaseController {
 	 */
 	private async findAll(
 		options: APIHandlerOptions<{
-			query: { projectId?: string } & PaginationQueryParameters;
+			query: ContributorGetAllRequestDto;
 		}>,
 	): Promise<APIHandlerResponse> {
-		if (options.query.projectId) {
-			const projectId = Number(options.query.projectId);
+		const { contributorName, hasHidden, orderBy, page, pageSize, projectId } =
+			options.query;
 
-			return {
-				payload: await this.contributorService.findAllByProjectId({
-					projectId,
-				}),
-				status: HTTPCode.OK,
-			};
-		}
+		const query = {
+			...(contributorName ? { contributorName } : {}),
+			...(hasHidden ? { hasHidden: Boolean(hasHidden) } : {}),
```

Comment:
In fact this line contains a bug: if we get `hasHidden` as `"false"` then this convertion will make it equal to `true`.
I didn't add add solution cause it looks a bit weird (it looks like `...(hasHidden ? { hasHidden: JSON.parse(hasHidden as unknown as string) as boolean } : {}),`). Also if we don't want to include hidden contributors we just shouldn't include `hasHidden` param to our query, so that's why it works.

So should I add this weird convertion just to for case, when we explicitly pass `hasHidden: false` to the query?

Reply:
That was so easy... Thanks